website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -4,37 +4,46 @@ id: "cloud-ci-job"
 description: "You can enable continuous integration (CI) to test every single change prior to deploying the code to production just like in a software development workflow."
 ---
 
-## Overview
 
-dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure jobs to run when new pull requests are opened against your dbt repo.
+dbt Cloud makes it easy to test every single code change you make prior to deploying that new logic into production. Once you've connected your [GitHub account](/docs/collaborate/git/connect-github), [GitLab account](/docs/collaborate/git/connect-gitlab), or [Azure DevOps account](/docs/collaborate/git/connect-azure-devops), you can configure continuous integration (CI) jobs to run when someone opens a new pull request in your dbt repository by setting up a webhook. For more information, refer to [Configuring a webhook](#configuring-a-webhook).
```

Comment:
"by setting up a webhook" is a bit confusing here - it is technically true in that that's how it's implemented, but you don't need to set up one of the newly released webhooks to make this happen. I would just drop that part of the sentence